### PR TITLE
Changes to steps involved for Tests verify_load_gpu_driver and verify_dhcp_client_timeout for Redhat

### PR DESCRIPTION
**Pull request by umfranci**
Following are the changes made:
- The name of the package for cuda driver on RHEL Images has been updated from 'cuda-drivers' to 'nvidia-driver-cuda', the details of which can be found here: [NVIDIA CUDA Installation Guide for Linux](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#removing-cuda-toolkit)
- The DHCP Client Timeout value for RHEL Images can be obtained from the path - 'etc/dhcp/dhclient.conf', similar to Debian and Suse Images. Hence making updates to use the same steps as these Distros.
- SKUs like **Standard_NV12ads_A10_v5** support **A10-8Q GPU**. Hence adding the same to the required list.

### CODE REVIEW

File: lisa/features/gpu.py [hunk 1]
```diff
@@ -201,7 +201,7 @@ def _install_cuda_driver(self, version: str) -> None:
                 "http://developer.download.nvidia.com/compute/cuda/"
                 f"repos/rhel{release}/x86_64/cuda-rhel{release}.repo"
             )
-            install_packages = ["cuda-drivers"]
+            install_packages = ["nvidia-driver-cuda"]
```
Comment (squirrelsc) [inline lisa/features/gpu.py:204]: Make sure it applies to all redhat versions.

Comment (umfranci) [reply lisa/features/gpu.py:204]: Sure @squirrelsc, have tested this with multiple versions, the list of which is as such:

_redhat:rhel:7lvm-gen2:latest, redhat:rhel:85-gen2:latest, redhat:rhel:86-gen2:latest, redhat:rhel:8-lvm-gen2:latest, redhat:rhel:90-gen2:latest, redhat:rhel:91-gen2:latest, redhat:rhel:93-gen2:latest, redhat:rhel:92-gen2:latest_

and some older versions too..

**Comment by umfranci** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by squirrelsc** [comment]
@LiliDeng LGTM

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**